Libplanet/Blocks/BlockHeader.cs

```diff
@@ -59,6 +64,10 @@ namespace Libplanet.Blocks
         /// Goes to the <see cref="TxHash"/>.</param>
         /// <param name="hash">The hash of the <see cref="Block{T}"/>.
         /// Goes to the <see cref="Hash"/>.</param>
+        /// <param name="preEvaluationHash">The hash of pre-commit block.
+        /// using for check validate.</param>
```

Comment:
Follow the consistent docs to the Block<T>() constructor's parameter.

```suggestion
        /// <param name="preEvaluationHash">The hash derived from the block <em>except of</em>
        /// <paramref name="evaluationDigest"/> (i.e., without action evaluation).
        /// Used for <see cref="Validate"/> checking <paramref name="nonce"/>.
        /// </param>
```